packages/babel-plugin-jest-hoist/package.json

```diff
@@ -34,7 +34,6 @@
     "@prettier/sync": "^0.5.0",
     "@types/babel__template": "^7.0.2",
     "@types/node": "*",
-    "@types/prettier": "^3.0.0",
```

Comment:
this is a stub now - prettier@3 ships with types

Reply:
I've opened https://github.com/jestjs/jest/pull/15041 to tackle this, to help reduce the diff here - seems like you might have some work ahead with the general refresh; let me know if there's anything else I can do to help 😅